packages/libs/lambda-at-edge/tests/default-handler/default-build-manifest-with-404.json

```diff
@@ -40,7 +40,6 @@
     "html": {
       "nonDynamic": {
         "/": "pages/index.html",
-        "/index": "pages/index.html",
```

Comment:
"/index" is not a valid route for the current Next.js app builds (only "/" is). It was inadvertently causing us to pass the basepath tests as "/basePath" redirected to /index.html due to this. So I removed it.